lib/MiLight/MiLightClient.h

```diff
@@ -11,7 +11,8 @@
 //#define DEBUG_PRINTF
 
 #define MILIGHT_DEFAULT_RESEND_COUNT 10
-
+//Used to determine close to white
+#define RGB_BOUNDARY 40
```

Comment:
Not a big deal, but could we call this `RGB_WHITE_BOUNDARY` or something along those lines?